### ISSUE TITLE
Creating CustomDataObject

### ISSUE THREAD

**Issue by kristines1119** [EncompassRest/EncompassRest#404]
Hello,

Can you please advise if I am doing the correct thing? I am trying to create a new custom data object to encompass

string fileName = "GSAFile.json";
var customDataObj = new CustomDataObject(fileName, content);
await _client.Company.GlobalCustomDataObjects.CreateOrReplaceCustomDataObjectAsync(customDataObj);

Thank you.

**Comment by TylerBrinkley** [comment]
Yes, that looks good. You can also look at the unit test here for guidance. https://github.com/EncompassRest/EncompassRest/blob/master/src/EncompassRest.Tests/GlobalCustomDataObjectsTests.cs

**Comment by kristines1119** [comment]
@TylerBrinkley Thank you. I tried the unit test above. It ran with no exceptions. 
If I check the Encompass Input form builder for this Custom Data Object, I do not see it.  Is this where it would show up if it were created appropriately?

Thanks.

**Comment by wconnors77** [comment]
@kristines1119 Hi, wondering if this was working for you now?
